test/e2e/main_test.go

```diff
@@ -31,8 +31,8 @@ func TestMain(m *testing.M) {
 	usr, _ := user.Current()
 	kubeconfig := flag.String("kubeconfig", filepath.Join(usr.HomeDir, "/.kube/config"),
 		"kube config path, e.g. $HOME/.kube/config")
-	gsimage := flag.String("gameserver-image", "gcr.io/agones-images/cpp-simple-server:0.2",
-		"gameserver image to use for those tests, gcr.io/agones-images/cpp-simple-server:0.2")
+	gsimage := flag.String("gameserver-image", "gcr.io/agones-images/udp-server:0.3",
```

Comment:
@Kuqd I updated this to make in IDE testing easier - and to match the `Makefile` - do you see any issue?

Reply:
good catch it was a mistake from me.